### ISSUE TITLE
Is it possible to adjust the layout to the bottom?

### ISSUE THREAD

**Issue by yil3** [nanotee/sqls.nvim#25]

**Comment by 4DRIAN0RTIZ** [comment]
Of course, although it cannot be done directly by overwriting the settings in the setup.

We can modify the source code:

### Installation Paths by Plugin Manager

| Plugin Manager   | Installation Path                                                      |
|------------------|-------------------------------------------------------------------------|
| **Packer**       | `~/.local/share/nvim/site/pack/packer/start/sqls.nvim/lua/sqls/`        |
| **Vim-Plug**     | `~/.local/share/nvim/plugged/sqls.nvim/lua/sqls/`                       |
| **Lazy.nvim**    | `~/.local/share/nvim/lazy/sqls.nvim/lua/sqls/`                          |

Once you're in the correct folder, follow the steps below:

1. Open the `commands.lua` file:

    ```bash
    nvim commands.lua
    ```

2. Find the `make_show_results_handler` function:

    ```lua
    function make_show_results_handler(...)
    ```

3. Change the line containing:

    ```lua
    vim.cmd(('%s pedit %s'):format(mods or '', tempfile))
    ```

4. Replace it with:

    ```lua
    vim.cmd(('botright %s pedit %s'):format(mods or '', tempfile))
    ```

And ready, keep in mind that if it is updated the change will no longer be seen, but knowing this method you could make some script or something that allows you to change it, or manually.

**Comment by nanotee** [comment]
This plugin follows the Vim philosophy of using modifiers to alter the behavior of commands, so you should be able to write `:botright SqlsExecuteQuery` for example. Applying your own patches to plugins can be a bit of a hassle, so I'd suggest creating a wrapper command or a mapping if you always want the same behavior.

**Comment by 4DRIAN0RTIZ** [comment]
Thanks for the aswer!
